hyperspy/_signals/signal1d.py

```diff
@@ -1490,14 +1493,23 @@ def estimate_peak_width(
         -------
         width or [width, left, right], depending on the value of
         `return_interval`.
-        """
 
+        """
         if show_progressbar is None:
             show_progressbar = preferences.General.show_progressbar
         self._check_signal_dimension_equals_one()
         if not 0 < factor < 1:
             raise ValueError("factor must be between 0 and 1.")
 
+        from hyperspy.misc.config_dir import os_name
+
+        if parallel and os_name == "windows":  # pragma: no cover
```

Comment:
in case `parallel=None`, it will pick up the value of `preferences.General.parallel` and in this case, it would be better to set parallel=False.

Reply:
Good idea, will edit